tokio-sync/src/watch.rs

```diff
@@ -60,14 +60,16 @@ use crate::task::AtomicWaker;
 use core::task::Poll::{Pending, Ready};
 use core::task::{Context, Poll};
 use fnv::FnvHashMap;
-use futures_core::ready;
 use futures_util::future::poll_fn;
-use futures_util::pin_mut;
 use std::ops;
 use std::sync::atomic::AtomicUsize;
 use std::sync::atomic::Ordering::SeqCst;
 use std::sync::{Arc, Mutex, RwLock, RwLockReadGuard, Weak};
 
+#[cfg(feature = "async-traits")]
+use futures_core::ready;
```

Comment:
why this change?

Reply:
It is a change to fix the warnings mentioned in https://github.com/tokio-rs/tokio/pull/1416#issuecomment-519820575.
https://dev.azure.com/tokio-rs/Tokio/_build/results?buildId=1874
```text
error: unused import: `futures_core::ready`
  --> tokio-sync/src/watch.rs:63:5
   |
63 | use futures_core::ready;
   |     ^^^^^^^^^^^^^^^^^^^
   |
   = note: `-D unused-imports` implied by `-D warnings`

error: unused import: `futures_util::pin_mut`
  --> tokio-sync/src/watch.rs:65:5
   |
65 | use futures_util::pin_mut;
   |     ^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Ahh I see now 👍